src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -190,31 +343,3 @@ export const NewSearchBarInputQueryRenderer: FC<NewSearchBarInputQueryRendererPr
     />
   )
 }
-
-/**
- * Displays during SSR render, but once mounted is swapped out with
- * QueryRenderer below.
- */
-const StaticSearchContainer: FC<{ searchQuery: string } & BoxProps> = ({
-  searchQuery,
-  ...rest
-}) => {
-  const { t } = useTranslation()
-  return (
-    <>
-      <Box display={["block", "none"]} {...rest}>
-        <SearchInputContainer
-          placeholder={searchQuery || t`navbar.searchArtsy`}
-          defaultValue={searchQuery}
-        />
-      </Box>
-
-      <Box display={["none", "block"]} {...rest}>
-        <SearchInputContainer
-          placeholder={searchQuery || t`navbar.searchBy`}
-          defaultValue={searchQuery}
-        />
-      </Box>
-    </>
-  )
-}
```

Comment:
So there's definitely a content shift happening that needs to get fixed before this get merged. Not sure if this is related but beware.

Reply:
Yeah it is fixed already, review app was staled a bit